From d6cc84d452e4a7a194578eceb24eaf6e183e01d4 Mon Sep 17 00:00:00 2001
From: Gilles Dartiguelongue <gilles.dartiguelongue@esiee.org>
Date: Sun, 11 Mar 2012 14:52:40 +0100
Subject: [PATCH 3/3] Add missing link to libm

This used to cause problems for gold linker while libemap was a static
library. Now libtool is smart enough to pick libm from the intermediate
library but it is still better to be explicit.
---
 src/time/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/time/Makefile.am b/src/time/Makefile.am
index 6ff332f..02c12db 100644
--- a/src/time/Makefile.am
+++ b/src/time/Makefile.am
@@ -15,7 +15,7 @@ INCLUDES += $(GST_TOOL_CFLAGS) $(DBUS_CFLAGS)
 CLEANFILES = $(GST_TOOL_CLEANFILES)
 EXTRA_DIST = $(GST_TOOL_EXTRA_DIST)
 
-time_admin_LDADD = $(GST_TOOL_LIBS) $(DBUS_LIBS) ./e-map/libemap.la
+time_admin_LDADD = -lm $(GST_TOOL_LIBS) $(DBUS_LIBS) ./e-map/libemap.la
 time_admin_DEPENDENCIES = $(GST_TOOL_DEPENDENCIES) ./e-map/libemap.la
 time_admin_SOURCES = \
 	time-tool.c	time-tool.h \
-- 
1.7.8.5

